### DISCUSSION TITLE
nginx rewrite的last关键词，apisix有没有方法实现？

### DISCUSSION THREAD

**Discussion by Lewisyixin** [apache/apisix#7638] [Q&A]
研究了一下apisix和nginx的配置，如果没理解错的话，
apisix的redirect插件和nginx rewrite中的redirect和permanent两个跳转等价
apisix的proxy-rewrite插件约等于 nginx rewrite的break跳转(区别在于nginx break之后可以访问静态资源，但是apisix作为网关路由只能转给上游)
但是目前并未发现rewrite last的实现方式(不改变浏览器url的情况下内部重定向到其他路由)

**Comment by tzssangglass** [comment]
> 内部重定向到其他路由

yes, the request comes in and only one route match is supported.

**Reply by tokers** [reply]
@Lewisyixin You have to write some Lua codes with the `ngx.exec` API in current stage.

**Reply by Lewisyixin** [reply]
Ok I see
